practice/e-commerce-website/src/components/CartSummary/CartSummary.tsx

```diff
@@ -0,0 +1,51 @@
+/* Import constants*/
+import { COMPONENT_SIZES, TEXT_VARIANTS } from '../../constants';
+/* Import components */
+import Text from '../Text/Text';
+import Divider from '../Divider/Divider';
+import Button from '../Button/Button';
+import InputField from '../InputField/InputField';
+import { DiscountIcon, RightArrowIcon } from '../Icon';
+/* Import helpers */
+import { calcDiscount, calcSummaryTotal } from '../../helpers/calcPrice';
+/* Import CSS */
+import './CartSummary.css';
+
+const CartSummary = ({
+  subtotal,
+  discount,
+  deliveryFee
+}: {
+  subtotal: number;
+  discount: number;
+  deliveryFee: number;
+}) => {
+  return (
+    <div className="cart-summary">
+      <Text variant={TEXT_VARIANTS.name} content={'Order Summary'} className="cart-summary-title" />
```

Comment:
should define children props for content
<Text variant={TEXT_VARIANTS.name} className="cart-summary-title">Order Summary</Text>